<!-- ps4_control_step.launch-->
<launch>
 <!-- Arduino_Serial Node-->
   <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
   <param name="port" value="/dev/ttyACM0"/>
   <param name="baud" value="57600"/>
   </node>
   
 <!-- joy node -->
  <node pkg="joy" type="joy_node" name="ps4_joy" respawn="true">
  <param name="dev" type="string" value="/dev/input/js1" />
  <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <node pkg="po_ps4_step_motor" type="ps4_control_step" name="teleop">
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="2" type="double"/>
  <param name="scale_angular" value="2" type="double"/>
  </node>



</launch>